tests/common/Configuration.cs

```diff
@@ -682,25 +674,6 @@ public static string MlaunchPath {
 		}
 
 #if !XAMMAC_TESTS
-		public static void AssertRuntimeIdentifierAvailable (ApplePlatform platform, string runtimeIdentifier)
```

Comment:
All this method are dependent of nunit. I moved them to a partial class and that way we can use the other ones in a XUnit project.